lib/sponsors_web/router.ex

```diff
@@ -13,6 +13,10 @@ defmodule SponsorsWeb.Router do
     pipe_through :api
 
     get "/health_check", HealthCheckController, :health_check
+
+    scope "/webhooks", Webhooks do
+      post "/stripe", StripeController, :create
```

Comment:
We'll have a webhook for mailchimp eventually so I wanted to get this namespace started.